src/relstorage/_util.py

```diff
@@ -25,6 +25,10 @@
 import os
 
 import logging
+
+from functools import partial
+from hashlib import md5 as md5_original
+
```

Comment:
I'd prefer to see this code in `_compat.py`. That module is for papering over differences between systems.